TicTacToe/GamePlay.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+
+namespace TicTacToe
+{
+    public class GamePlay
+    {
+        private int _player;
+        private GameStatus _gameStatus;
+        private readonly Board _board;
+        private readonly int[,] _boardArray;
```

Comment:
you have both a board and a boardArray which I find confusing, it looks like all you use the board for is to get the boardArray?